main.go

```diff
@@ -101,9 +108,31 @@ func updateBenchmarks() {
 	}
 }
 
-func runBenchmarks() (Set, error) {
+func versionRequired(required, tag string) bool {
+	if required == "" {
+		return true
+	}
+	tagVer, _ := semver.Make(trimTagVersion(tag))
+
+	if strings.HasPrefix(required, ">=") {
+		requiredVer, _ := semver.Make(trimTagVersion(strings.TrimLeft(required, ">=")))
+		return tagVer.GTE(requiredVer)
+	}
+	if strings.HasPrefix(required, ">") {
+		requiredVer, _ := semver.Make(trimTagVersion(strings.TrimLeft(required, ">")))
+		return tagVer.GT(requiredVer)
+	}
+	requiredVer, _ := semver.Make(trimTagVersion(required))
+	return tagVer.Equals(requiredVer)
+}
+
+func runBenchmarks(tagVersion string, isTag bool) (Set, error) {
```

Comment:
it doesn't seem that you need 2 different parameters. you can use `version string` with an empty string meaning "not a tag".

Reply:
Good advice. Updated.